testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/AgentTestRunner.java

```diff
@@ -105,31 +92,13 @@
               .addTextMapPropagator(HttpTraceContext.getInstance())
               .build());
     }
-    OpenTelemetrySdk.getGlobalTracerManagement().addSpanProcessor(TEST_WRITER);
     TEST_TRACER = OpenTelemetry.getGlobalTracer("io.opentelemetry.auto");
```

Comment:
maybe we need to add `opentelemetry-api-beta` as default instrumentation for all tests? lots of tests use `TraceUtils.runUnderTrace()`

Reply:
Ah I guess that's because previously SDK and API were in the same classloader without bridging so it wasn't necessary, now we do need it. Sounds good